exercises/twelve-days/twelve_days_test.exs

```diff
@@ -0,0 +1,329 @@
+      assert TwelveDays.count(11) == "eleven"
+    end
+
+    @tag :pending
+    test "12 returns twelve" do
+      assert TwelveDays.count(12) == "twelve"
+    end
+  end
+
+  describe "ordinal" do
+    @tag :pending
+    test "1 returns first" do
+      assert TwelveDays.ordinal(1) == "first"
+    end
+
+    @tag :pending
+    test "2 returns second" do
+      assert TwelveDays.ordinal(2) == "second"
+    end
+
+    @tag :pending
+    test "3 returns third" do
+      assert TwelveDays.ordinal(3) == "third"
+    end
+
+    @tag :pending
+    test "4 returns fourth" do
+      assert TwelveDays.ordinal(4) == "fourth"
+    end
+
+    @tag :pending
+    test "5 returns fifth" do
+      assert TwelveDays.ordinal(5) == "fifth"
+    end
+
+    @tag :pending
+    test "6 returns sixth" do
+      assert TwelveDays.ordinal(6) == "sixth"
+    end
+
+    @tag :pending
+    test "7 returns seventh" do
+      assert TwelveDays.ordinal(7) == "seventh"
+    end
+
+    @tag :pending
+    test "8 returns eighth" do
+      assert TwelveDays.ordinal(8) == "eighth"
+    end
+
+    @tag :pending
+    test "9 returns ninth" do
+      assert TwelveDays.ordinal(9) == "ninth"
+    end
+
+    @tag :pending
+    test "10 returns tenth" do
+      assert TwelveDays.ordinal(10) == "tenth"
+    end
+
+    @tag :pending
+    test "11 returns eleventh" do
+      assert TwelveDays.ordinal(11) == "eleventh"
+    end
+
+    @tag :pending
+    test "12 returns twelfth" do
+      assert TwelveDays.ordinal(12) == "twelfth"
+    end
+  end
+
+  describe "gift" do
+    @tag :pending
+    test "partridge" do
+      assert TwelveDays.gift(1) == "a Partridge in a Pear Tree"
+    end
+
+    @tag :pending
+    test "turtle doves" do
+      assert TwelveDays.gift(2) == "two Turtle Doves"
+    end
+
+    @tag :pending
+    test "french hens" do
+      assert TwelveDays.gift(3) == "three French Hens"
+    end
+
+    @tag :pending
+    test "calling birds" do
+      assert TwelveDays.gift(4) == "four Calling Birds"
+    end
+
+    @tag :pending
+    test "gold rings" do
+      assert TwelveDays.gift(5) == "five Gold Rings"
+    end
+
+    @tag :pending
+    test "geese-a-laying" do
+      assert TwelveDays.gift(6) == "six Geese-a-Laying"
+    end
+
+    @tag :pending
+    test "swans-a-swimming" do
+      assert TwelveDays.gift(7) == "seven Swans-a-Swimming"
+    end
+
+    @tag :pending
+    test "maids-a-milking" do
+      assert TwelveDays.gift(8) == "eight Maids-a-Milking"
+    end
+
+    @tag :pending
+    test "ladies dancing" do
+      assert TwelveDays.gift(9) == "nine Ladies Dancing"
+    end
+
+    @tag :pending
+    test "lords-a-leaping" do
+      assert TwelveDays.gift(10) == "ten Lords-a-Leaping"
+    end
+
+    @tag :pending
+    test "pipers piping" do
+      assert TwelveDays.gift(11) == "eleven Pipers Piping"
+    end
+
+    @tag :pending
+    test "drummers drumming" do
+      assert TwelveDays.gift(12) == "twelve Drummers Drumming"
+    end
+  end
+
+  describe "verse" do
```

Comment:
I don't think we need tests for every verse in here. Maybe we can test the first verse, the last verse, and one more in the middle?